src/app/helps/[id]/page.jsx

```diff
@@ -291,7 +300,7 @@ export default async function HelpPage({ params }) {
                   </Grid>
                 </div>
                 <Divider
-                  vertical
```

Comment:
Unassigned boolean prop is equivalent of `vertical=true`

Reply:
When using only vertical a warning was raised in the browser console suggesting this approach.

Reply:
Ah, the DOM couldn't recognize it because the property is deprecated. It's been changed to **orientation**.

Nice catch 👍🏻